app/views/annotations/_annotation_modal.html.erb

```diff
@@ -49,7 +49,10 @@
       <% end %>
 
       <section class='dialog-actions'>
-        <%= submit_tag title, id:'submit_annotation', data: submit_btn_data %>
+        <%= submit_tag (change_one_option ? title_all : title), id:'submit_annotation', data: submit_btn_data %>
+        <% if change_one_option %>
+          &nbsp;&nbsp;<%= submit_tag title_one, id: 'submit_this_annotation', name: 'change_one' %>
```

Comment:
After looking at this more, I think having two buttons is a bit too cumbersome (especially since the labels are quite long). Instead, please keep the old submit button, and add a checkbox input to the form that says something like "Modify all occurrences of this annotation".